code/modules/surgery/organs/tongue.dm

```diff
@@ -36,6 +36,8 @@
 		/datum/language/vox, //SKYRAT EDIT - customization - extra languages
 		/datum/language/nekomimetic,
 		/datum/language/neorusskya,  //SKYRAT EDIT - customization - extra languages
+		/datum/language/arabic,  //SKYRAT EDIT - customization - extra languages
```

Comment:
should type it as the name it has (Xerxian) instead of referring to it in-code as arabic